packages/components/pie-checkbox/src/checkbox.scss

```diff
@@ -179,13 +184,18 @@
     background-color: white;
 }
 
+.c-checkbox-tick[isRtl]:after {
+    left: unset;
+    right: 14%;
+}
+
 .c-checkbox {
```

Comment:
NIT – I'd expect the class styling to be in HTML order. So `c-checkbox` to be before `c-checkbox-tick` and other child element styling.

Reply:
Since there is nesting in `.c-checkbox {}` our linting rules does not allow .c-checkbox to be before `.c-checkbox-tick`.

The error:
```
✖  Expected selector ".c-checkbox-tick" to come before selector ".c-checkbox:hover .c-checkbox-tick"  no-descending-specificity
```

Reply:
Ok - maybe something can look at later when we switch the styles to use classmap